specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/machineLearningServices.json

```diff
@@ -0,0 +1,4768 @@
+                },
+                "agentVmSize": {
+                  "description": "Agent virtual machine size",
+                  "type": "string"
+                },
+                "clusterPurpose": {
+                  "description": "Intended usage of the cluster",
+                  "type": "string",
+                  "default": "FastProd",
+                  "enum": [
+                    "FastProd",
+                    "DenseProd",
+                    "DevTest"
+                  ],
+                  "x-ms-enum": {
+                    "name": "ClusterPurpose",
+                    "modelAsString": true
+                  }
+                },
+                "sslConfiguration": {
+                  "description": "SSL configuration",
+                  "$ref": "#/definitions/SslConfiguration"
+                },
+                "aksNetworkingConfiguration": {
+                  "description": "AKS networking configuration for vnet",
+                  "$ref": "#/definitions/AksNetworkingConfiguration"
+                },
+                "loadBalancerType": {
+                  "description": "Load Balancer Type",
+                  "type": "string",
+                  "default": "PublicIp",
+                  "enum": [
+                    "PublicIp",
+                    "InternalLoadBalancer"
+                  ],
+                  "x-ms-enum": {
+                    "name": "LoadBalancerType",
+                    "modelAsString": true
+                  }
+                },
+                "loadBalancerSubnet": {
+                  "description": "Load Balancer Subnet",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "AKS"
+    },
+    "AmlCompute": {
+      "description": "An Azure Machine Learning compute.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "type": "object",
+              "description": "AML Compute properties",
+              "properties": {
+                "osType": {
+                  "description": "Compute OS Type",
+                  "type": "string",
+                  "default": "Linux",
+                  "enum": [
+                    "Linux",
+                    "Windows"
+                  ],
+                  "x-ms-enum": {
+                    "name": "OsType",
+                    "modelAsString": true
+                  }
+                },
+                "vmSize": {
+                  "description": "Virtual Machine Size",
+                  "type": "string"
+                },
+                "vmPriority": {
+                  "description": "Virtual Machine priority",
+                  "type": "string",
+                  "enum": [
+                    "Dedicated",
+                    "LowPriority"
+                  ],
```

Comment:
Same as above, could we move these properties to `AmlCompute` instead of using this anoynmous object?